js/vm/documentlist.js

```diff
@@ -16,8 +16,11 @@ export default class DocumentList {
 
   load(name) {
     this.lecture = name;
-    config.getJSON('/data/lectures/' + encodeURIComponent(name) + '/documents')
-      .done(data => this.documents = data.map(d => new Document(d)));
+    if (name)
+      config.getJSON('/data/lectures/' + encodeURIComponent(name) + '/documents')
+        .done(data => this.documents = data.map(d => new Document(d)));
+    else
+      this.documents = [];
```

Comment:
I would remove the else and move ```this.documents = [];``` before the if. Otherwise it is undefined before the callback gets called.